.gitignore

```diff
@@ -13,3 +13,6 @@ npm-debug.log
 # coverage
 .nyc_output
 coverage
+
+# editors
+.idea/
```

Comment:
This can be managed by using https://help.github.com/articles/ignoring-files/#create-a-global-gitignore